### PR TITLE
Make API key secret

### PR DESCRIPTION
**Pull request by TheChrisVela**
I have already rotated my API keys

### CODE REVIEW

File: app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java [hunk 1]
```diff
@@ -26,7 +27,7 @@ public class RestClient extends OAuthBaseClient {
 	public static final BaseApi REST_API_INSTANCE = FlickrApi.instance(FlickrApi.FlickrPerm.WRITE); // Change this
 	public static final String REST_URL = "twitter.comp";
```
Comment (ChrisGuzman) [inline app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java:28]: Hey this is wrong

File: app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java [hunk 2]
```diff
@@ -26,7 +27,7 @@ public class RestClient extends OAuthBaseClient {
...
 	public static final String REST_CONSUMER_KEY = CONSUMER_KEY;
-	public static final String REST_CONSUMER_SECRET = "EeyJ4vEZN3al7c0C13bMwAY3pGc2RASrampYtvJvnX1kLDHKJf";
+	public static final String REST_CONSUMER_SECRET = CONSUMER_SECRET;
 
 	// Landing page to indicate the OAuth flow worked in case Chrome for Android 25+ blocks navigation back to the app.
 	public static final String FALLBACK_URL = "https://codepath.github.io/android-rest-client-template/success.html";
```
Comment (TheChrisVela) [inline app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java:33]: I like that you used a static var for this!

File: app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java [hunk 3]
```diff
@@ -26,7 +27,7 @@ public class RestClient extends OAuthBaseClient {
 	public static final BaseApi REST_API_INSTANCE = FlickrApi.instance(FlickrApi.FlickrPerm.WRITE); // Change this
 	public static final String REST_URL = "twitter.comp";
 	public static final String REST_CONSUMER_KEY = CONSUMER_KEY;
-	public static final String REST_CONSUMER_SECRET = "EeyJ4vEZN3al7c0C13bMwAY3pGc2RASrampYtvJvnX1kLDHKJf";
+	public static final String REST_CONSUMER_SECRET = CONSUMER_SECRET;
```
Comment (TheChrisVela) [inline app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java:30]: Great job with the import!

File: app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java (hunk 1 above)
Comment (TheChrisVela) [inline app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java]:
```suggestion
	public static final String REST_URL = "twitter.com";
```
I think you have a typo.

**Review by TheChrisVela — COMMENTED** [review]
I think you have some bugs.

File: app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java (hunk 2 above)
Comment (TheChrisVela) [reply app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java:33]: Thank you!

I wanted to do something like this

```kotlin
val secret = CONSUMER_SECRET
```

**Pull request closed without merging**